### ISSUE TITLE
Unknown Error: Could not format code. Full error:  Invalid regular expression: /^(*)(?!⋮$)/gm: Nothing to repeat

### ISSUE THREAD

**Issue by liuwa666** [0h7z/vscode-julia-format#6]
```console
Julia Version 1.9.4
Commit 8e5136fa29 (2023-11-14 08:46 UTC)
Build Info:
  Official https://julialang.org/ release
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: 16 × AMD Ryzen 9 5900HX with Radeon Graphics
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.6 (ORCJIT, znver3)
  Threads: 1 on 16 virtual cores
```
```console
Julia Version 1.9.4
Commit 8e5136fa297 (2023-11-14 08:46 UTC)
Build Info:
  Official https://julialang.org/ release
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: 16 × AMD Ryzen 9 5900HX with Radeon Graphics
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.6 (ORCJIT, znver3)
  Threads: 1 on 16 virtual cores
```

@Heptazhou

**Comment by Heptazhou** [comment]
Oops.

**Comment by Heptazhou** [comment]
Fixed in v1.2.4. Thanks for reporting!

**Comment by liuwa666** [comment]
> Fixed in v1.2.4. Thanks for reporting!
thanks for your selfless dedication

**Comment by liuwa666** [comment]
> Fixed in v1.2.4. Thanks for reporting!

thanks for your selfless dedication
